src/Environment.php

```diff
@@ -873,7 +888,7 @@ public function getUnaryOperators(): array
     /**
      * @internal
      *
-     * @return array<string, array{precedence: int, class: class-string<AbstractBinary>, associativity: ExpressionParser::OPERATOR_*}>
+     * @return array<string, array{precedence: int, class: class-string<AbstractExpression>, associativity: ExpressionParser::OPERATOR_*}>
```

Comment:
Not always an AbstractBinary since CoreExtension::getOperators returns `NullCoalesceExpression` which is not extending AbstractBinary.

I did the same for Unary just in case.